ImguiMapEditor/Application/CallbackMediator.cpp

```
@@ -144,14 +170,10 @@ void CallbackMediator::wireTabCallbacks(Context &ctx) {
   });
 
   // Hotkey map menu
+  // Dependencies were injected once in wireAll(); dialog auto-discovers the
+  // active map via MapTabManager, so show() needs no parameters.
   ctx.hotkey->setEditTownsCallback([ctx]() {
```

Clang (14.0.6)

[note] nesting level increased to 1

(clang)